components/brave_wallet_ui/components/buy-with-button/style.ts

```diff
@@ -0,0 +1,74 @@
+import styled from 'styled-components'
+
+import CaratDownIcon from '../../assets/svg-icons/select-down-icon.svg'
```

Comment:
are we able to load these icons in a non-SVG format? SVG seems a bit risky since it allows script execution (unless that is blocked via CSP or similar): https://developer.mozilla.org/en-US/docs/Web/SVG/Element/script

if we need SVG i think we should have some kind of hardening so SVG scripts can't run in webui, cc @petemill

Reply:
i realize these are "trusted" icons but i'd rather not have to carefully audit each SVG update if possible

Reply:
I have created an issue to fix this problem since SVGs are currently used in Wallet UI extensively. https://github.com/brave/brave-browser/issues/22296

Reply:
@muliswilliam where else do we use SVGs? according to https://bravesoftware.slack.com/archives/C8MP8ME4C/p1650576580433059?thread_ts=1650575479.220099&cid=C8MP8ME4C we shouldn't be since they aren't supported on iOS.

btw - i think our CSP does block inline script execution from SVGs but there are probably other attacks that are possible with malicious SVGs.

Reply:
@diracdeltas SVGs are used in most places we are displaying icons in the wallet(I can add a few references for this if need be). The line in question is for Web UI and iOS wallet team can use the format of their choice.

Reply:
@diracdeltas that comment was in reference to outside generated icons that we use for assets that we include. 
For icons that we author ourselves and that are in our source tree we have svgs on desktop.